src/helm/benchmark/scenarios/vision_language/bingo_scenario.py

```diff
@@ -43,7 +43,7 @@ class BingoScenario(Scenario):
     Paper: https://arxiv.org/abs/2311.03287
     """
 
-    BINGO_HUGGINGFACE_DATASET_NAME: str = "PahaII/Bingo"
+    BINGO_HUGGINGFACE_DATASET_URL: str = "https://huggingface.co/datasets/PahaII/Bingo/resolve/main"
```

Comment:
Does this have any affect if I already have the HuggingFace dataset cached?

Reply:
Nope, it will load the same cached data